### ISSUE TITLE
Text to speech audio on iOS not working

### ISSUE THREAD

**Issue by raynergit** [omnivore-app/omnivore#4416]
Over the last week or so, text-to-speech stopped working on my device. I usually use Alloy for audio generation, and now that voice, along with most other English voices, display this upon attempting to play text to speech:

"There was an error playing back your audio."

Brandon, Christopher and Cora seem to work, but Alloy seems to have consistent issues. There may be other voices with issues, but I have not tried all of them yet. I've logged out, logged back in, and deleted the app. I'm on iOS 17.6.1. Thank you!

**Comment by glidergold** [comment]
I searched to see if anyone else was experiencing this issue and found this thread. It sounds like the more sophisticated, natural voices are giving me the same error, whereas the more computer-sounding voices still work.

**Comment by raynergit** [comment]
Super unfortunate! I plug Omnivore and encourage people to donate wherever I can but definitely finding comfort in Substack until we can get this resolved.

**Comment by raynergit** [comment]
Super unfortunate! I accidentally marked this as resolved, but it's definitely not. Reopening with a new comment.

**Comment by markjmidlick** [comment]
Having the same issue as well

**Comment by saeedesmaili** [comment]
Same issue here for more than a week. Maybe we should open another issue if we can't re-open this one.

**Comment by adarsh** [comment]
I'm having this issue with the voice Shimmer. 

Any thoughts on how to debug this?

**Comment by JamesZlearner** [comment]
same issue here

**Comment by paulschm** [comment]
same here!

**Comment by wrbijlsma** [comment]
Also experiencing the issue, now with all voices

**Comment by saeedesmaili** [comment]
Yes, I think they shut down the TTS service completely.

**Comment by raynergit** [comment]
Yeah, very sad about this. Some of the best TTS speech I've heard was on Omnivore, but they've joined another company, ElevenLabs, and their voices sound robotic to me, unfortunately. The app we've been using will be shutting down:

"Next, all Omnivore users will be able to export their information from the service through November 15 2024, after which all information will be deleted. 

Lastly, ElevenLabs is committed to the developer community and the Omnivore codebase will remain 100% open-source for all users. This decision ensures that the broader development community can continue to build upon and improve Omnivore’s technology."

Notes on exporting: https://docs.omnivore.app/using/exporting.html

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
